src/tools/gditest.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+CHDMAN="/home/nathanh/github/mame/chdman"
```

Comment:
Was this line, if not file, supposed to be included?

Reply:
The file is not required. It was used to test compatibility with existing .gdi support. I've removed it in 91af4f306760585ffc48a0bf302144cd81823cd8